spec/system/prior_authority/view_assessed_applications_spec.rb

```diff
@@ -94,6 +94,23 @@
     end
   end
 
+  context 'when application is sent back' do
+    let(:application) do
+      create(:prior_authority_application,
+             :full,
+             status: 'sent_back',
+             app_store_updated_at: 1.day.ago,
+             resubmission_requested: 1.day.ago,
+             resubmission_deadline: 12.days.from_now)
+    end
+
+    it 'shows expiry details' do
+      expect(page).to have_content 'Update needed'
+      expect(page).to have_content '£155.00 requested'
+      expect(page).to have_content 'Review the requests and resubmit your application by 3 April 2024'
```

Comment:
relative date with hardcoded expectation will break "tomorrow" I imagine.

I [had done this on my PR](https://github.com/ministryofjustice/laa-submit-crime-forms/pull/662/files#diff-0bf07e296341b5a0b28ba2a0ab37082a6ca7c44b68532c5fbbbf04c349546131R99-R118)

Reply:
Ah, but we already have a `before {travel_to arbitrary_fixed_date}` so this expectation is safe.